api.md

```diff
@@ -469,7 +528,81 @@ CREATE INDEX ON conditions(time, location) USING brin
 ```
 
 ---
+## detach_data_node() [](detach_data_node)
+
+Detach a data node from one hypertable or from all hypertables.
+
+Reasons for detaching a data node:
 
+- A data node should no longer be used by a hypertable and needs to be
+removed from all hypertables that use it
+- You want to have fewer data nodes for a distributed hypertable to
+partition across to other distributed hypertables in the database
+
+>:WARNING: This function *MUST* be run on an access node.
+
+#### Required Arguments [](detach_data_node-required-arguments)
+
+| Name        | Description                       |
+|-------------|-----------------------------------|
+| `node_name` | Local name of data node to detach |
+
+##### Parameter Description
+
+* `node_name` - Local name of the data node to detach from the
+  hypertable.
+
+#### Optional Arguments [](detach_data_node-optional-arguments)
+
+| Name         | Description                            |
+|--------------|----------------------------------------|
+| `hypertable` | Name of hypertable to detach node from |
+| `force`      | Force detach of the data node          |
+
+##### Parameter Description
```

Comment:
Work these descriptions into the table above.

Reply:
Done